README.md

```diff
@@ -1,4 +1,14 @@
 # NF1_SchwannCell_data_analysis
+## Repository Structure
+Here we analyze NF1 morphology data from the [NF1 Cell painting Data](https://github.com/WayScience/nf1_cellpainting_data) repo. Our analysis is categorized as follows:
+
+| Module | Purpose | Description |
+| :---- | :----- | :---------- |
+| [0.data_analysis](./0.data_analysis/) | Analyze NF1 Data | We find interesting patterns in the NF1 cell morphology data. |
+| [1.train_models](./1.train_models/) | Train NF1 Models | Optimize the NF1 model by training multiple models with a random search. |
+| [2.evaluate_models](./2.evaluate_models/) | Evaluate Final NF1 Model | After training the final NF1 model we evaluate model performance and model feature importances. |
+| [3.figures](./3.figures/) | Visualize Analysis Results| We then interpret our results visually. |
```

Comment:
```suggestion
| [3.figures](./3.figures/) | Generate manuscript figures | We interpret the results from the model by generating figures to include in the manuscript. |
```

Reply:
Should this module be "advertised" as containing only manuscript figures, or more generally, those results we want to interpret from the model? It may be fine to "advertise" this module as intended for model manuscript figures, since the manuscript will focus on interpreting the results of the model

Reply:
I believe it should be. We do not include any other "figures" in this module, so it makes sense to make it clear what the purpose of the module is for.

Reply:
Sounds good, I guess we can provide a link to the manuscript later